lib/rules/selector-attribute-quotes/index.js

```diff
@@ -36,25 +38,41 @@ function rule(expectation) {
 			}
 
 			parseSelector(ruleNode.selector, result, ruleNode, (selectorTree) => {
```

Comment:
```suggestion
			parseSelector(getRuleSelector(ruleNode), result, ruleNode, (selectorTree) => {
```

We'll need a new util here.

Reply:
> @doing-art Thank you for the pull request. It's almost there.
> 
> Can we add a test with a comment within a selector list please, e.g.:
> 
> ```
> {
>   code: 'a[target="_blank"], /* comment */ a { }',
>   fixed: 'a[target=_blank], /* comment */ a { }',
>   message: messages.rejected('_blank'),
>   line: 1,
>   column: 10,
> },
> ```
> 
> It'll fail because comments are stripped for the `selector` property of the AST. The full string is available in the `raws` property, and we'll want to pass this full string to the selector parser. ([Ref](https://astexplorer.net/#/gist/77cca53302eb6e016c2ef2516537594a/71abb107772e9c19798c5d5daf543bb2816ebf73)).
> 
> Let's create a `getRuleSelector` util based off [`getDeclarationValue`](https://github.com/stylelint/stylelint/blob/master/lib/utils/getDeclarationValue.js) and use it in the rule.

@jeddy3 Thanks for reviewing the PR. I made the requested change.